### ISSUE TITLE
 ClickHouseBatchRunnable exception

### ISSUE THREAD

**Issue by tellienne** [Altinity/clickhouse-sink-connector#705]
Hi!
When using the  connector 2.2.0-lt we see an error in the logs

```
2024-07-23 10:38:59.129 ERROR - ClickHouseBatchRunnable exception - Task(0)
java.util.UnknownFormatConversionException: Conversion = ')'
	at java.util.Formatter.parse(Formatter.java:2750) ~[?:?]
	at java.util.Formatter.format(Formatter.java:2671) ~[?:?]
	at java.util.Formatter.format(Formatter.java:2625) ~[?:?]
	at java.lang.String.format(String.java:4140) ~[?:?]
	at com.altinity.clickhouse.sink.connector.executor.ClickHouseBatchRunnable.processRecordsByTopic(ClickHouseBatchRunnable.java:307) ~[app.jar:?]
	at com.altinity.clickhouse.sink.connector.executor.ClickHouseBatchRunnable.run(ClickHouseBatchRunnable.java:188) [app.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) [?:?]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]

```
The error occurs during the first snapshot of a table in postgres and the snapshot cannot complete.
How can we find out what's wrong?

**Comment by subkanthi** [comment]
Hi @tellienne , can u try this image `altinityinfra/clickhouse-sink-connector:708-593c721c0197f07083ed78d42f80a760d52cafb9-lt`

**Comment by subkanthi** [comment]
![image](https://github.com/user-attachments/assets/20bb02df-bb41-470a-9292-03741c713fa0)

**Comment by subkanthi** [comment]
Fixed in #708
